File: task1/main.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Введите строку: ");
        string text = Console.ReadLine();

        string glas = "aeiouyаеёиоуыэюяAEIOUYАЕЁИОУЫЭЮЯ";
        string soglas = "bcdfghjklmnpqrstvwxzбвгджзйклмнпрстфхцчшщBCDFGHJKLMNPQRSTVWXZБВГДЖЗЙКЛМНПРСТФХЦЧШЩ";

        int countGlas = 0, countSog = 0;

        foreach (char ch in text)
        {
            if (glas.Contains(ch))
                countGlas++;
            else if (soglas.Contains(ch))
                countSog++;
        }

        if (countGlas > countSog)
            Console.WriteLine("да");
        else if (countGlas < countSog)
            Console.WriteLine("нет");
        else
            Console.WriteLine("одинаково");
    }

File: task1/main.swift
import Foundation

print("Введите строку: ", terminator: "")
if let text = readLine() {
    let glas = "aeiouyаеёиоуыэюяAEIOUYАЕЁИОУЫЭЮЯ"
    let soglas = "bcdfghjklmnpqrstvwxzбвгджзйклмнпрстфхцчшщBCDFGHJKLMNPQRSTVWXZБВГДЖЗЙКЛМНПРСТФХЦЧШЩ"
    
    var countGlas = 0, countSog = 0 // задаем счетчик для гласных и согласных
    
    // проходимся циклом по строке
    for char in text {
        if glas.contains(char) {
            countGlas += 1
        } else if soglas.contains(char) {
            countSog += 1
        }
    }
    
    // делаем вывод по условию задачи
    if countGlas > countSog {
        print("да")
    } else if countGlas < countSog {
        print("нет")
    } else {
        print("одинаково")
    }
}
